services/payments/controllers.go

```diff
@@ -111,7 +132,7 @@ func PrepareHandler(service *Service) handlers.AppHandler {
 			return handlers.WrapError(err, "failed to get attestation from nitro", http.StatusBadRequest)
 		}
 
-		resp.AttestationDocument = base64.StdEncoding.EncodeToString(attestationDocument)
+		w.Header().Add("X-Nitro-Attestation", base64.StdEncoding.EncodeToString(attestationDocument))
 		// Should be in QLDB in prepared state
```

Comment:
Would this comment be better on the prepare handler describing what is does?

Reply:
Yes. Moved it.